capstone-rs/src/instruction.rs

```diff
@@ -328,6 +345,64 @@ impl<'a> InsnDetail<'a> {
     }
 }
 
+impl<'a> InsnRegsAccess {
+    fn new(cs: csh, ins: &cs_insn) -> Self {
```

Comment:
Any function that can cause undefined behavior should be marked unsafe.

However, once you move the error checking (checking return value), then this shouldn't need to be marked unsafe.

Reply:
Have this return a `CsResult<Self>` instead